README.md

```diff
@@ -2,13 +2,21 @@
 
 This repo contains the software for the 'ALICE Electronic Logbook'. https://alice-logbook.cern.ch/
 
+## Running the application locally
+
+To run the application locally, call the start-dev.sh script. You can pass 3 arguments: 'All', 'frontend' or 'backend' to start the layers. Below is an example on how the script is called, please note that a **Bash environment** is required to run the script.
+
+```sh
+./start-dev.sh all
```

Comment:
This wont work, you are executing from the root of the project, missing the scripts directory

Reply:
The script has been adjusted to work from the root directory.